test/test_async_ttl_cache.py

```diff
@@ -9,10 +9,14 @@ class AsyncTTLCacheUnitTest(unittest.TestCase):
 
     @async_ttl_cache(ttl=3, maxsize=1)
     async def get_timestamp(self):
-        return time.time()
+        timestamp = time.time()
+        print(f"timestamp: {timestamp}")
+        return timestamp
```

Comment:
I think this inclusion wasn't intended.

Reply:
Oh, yep, you're right, this wasn't intentional.